scripts/dpanon.py

```diff
@@ -64,9 +75,13 @@ def parse_args(argv):
                         help='A directory to remove PII from')
     parser.add_argument('-od', '--out_dir',
                         help='PII removed output dir path')
-    parser.add_argument('-dt', '--datatype',
-                        choices=['surveys', 'video', 'audio', 'actigraphy', 'mri', 'eeg'],
-                        help='Datatype to remove PII (applies to -p, -i).')
+    parser.add_argument(
+            '-dt', '--datatype',
+            choices=['surveys', 'video', 'audio', 'actigraphy', 'mri', 'eeg'],
+            help='Datatype to remove PII (applies to -p, -i).')
+    parser.add_argument(
+            '-ptl', '--pii_table_loc', default=False,
```

Comment:
This `-ptl` should be changed. It is very easy to confuse between `1` and `L`.

Reply:
changed here - https://github.com/AMP-SCZ/dpanonymize/pull/13/commits/1fa28533854a33fb21520b8abf5601109be331af